File: Teast_Section/app/src/main/java/com/example/teast_section/MainActivity.java
package com.example.teast_section;

import android.content.DialogInterface;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import android.view.View;
import android.widget.Button;
import android.widget.RadioButton;
import android.widget.RadioGroup;
import android.widget.Toast;

import androidx.activity.EdgeToEdge;
import androidx.annotation.NonNull;
import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {
    Button button = findViewById(R.id.button);
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);

            Toolbar toolbar = findViewById(R.id.toolbar);
            setSupportActionBar(toolbar);

            button.setOnClickListener(new View.OnClickListener() {
                @Override
                public void onClick(View v) {
                    alertDialog().show();
                }
            });

//            RadioGroup radioGroup = findViewById(R.id.radiogroup);
//            radioGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener() {
//                @Override
//                public void onCheckedChanged(RadioGroup group, int checkedId) {
//                    RadioButton rd = findViewById(checkedId);
//                    if (rd.isChecked()) Toast.makeText(MainActivity.this ,rd.getText(),Toast.LENGTH_SHORT).show();
//
//                }
//            });


            return insets;
        });
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        MenuInflater menuInflater = new MenuInflater(this);
        menuInflater.inflate(R.menu.menulist,menu);

        return true;
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        int itemID = item.getItemId();
        if (itemID  == R.id.item1){
            Toast.makeText(this, item.toString(), Toast.LENGTH_SHORT).show();
        }
        return true;
    }

    AlertDialog alertDialog(){
        AlertDialog.Builder builder = new AlertDialog.Builder(this);
        builder.setMessage("Erase up");
        builder.setPositiveButton("ok", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {
                Toast.makeText(MainActivity.this, "ok", Toast.LENGTH_SHORT).show();
            }
        });

        builder.setNegativeButton("no", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {
                Toast.makeText(MainActivity.this, "no", Toast.LENGTH_SHORT).show();
            }
        });
        return builder.create();
    }
}

File: MyApplication3/app/src/main/java/com/example/myapplication3/MainActivity.kt
package com.example.myapplication3

import android.content.DialogInterface
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.widget.Toast
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AlertDialog
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.example.myapplication3.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
//     val myfragmentone = OneFragment()
//     val myfragmenttwo = TwoFragment()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        enableEdgeToEdge()
        setContentView(binding.root)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
//            initSpinner()
//            addnaf()
//            showDialog()
            binding.floating.setOnClickListener {
                Toast.makeText(this , "Click Me ",Toast.LENGTH_LONG).show()
            }
            WindowInsetsCompat.CONSUMED // Add this line to indicate that you've consumed the insets
        }

    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.action_bar,menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId){
            R.id.icon11 ->{
                Toast.makeText(this@MainActivity,"Icon 1",Toast.LENGTH_LONG).show()
            }
            R.id.icon22 ->{
                Toast.makeText(this@MainActivity,"Icon 2",Toast.LENGTH_LONG).show()
            }
            R.id.icon33 ->{
                Toast.makeText(this@MainActivity,"Icon 3",Toast.LENGTH_LONG).show()
            }
        }
        return super.onOptionsItemSelected(item)
    }
    private fun showDialog(){
        val builder = AlertDialog.Builder(this)
        builder.apply {
            setTitle("Dialog")
            setMessage("Massage")
            setPositiveButton("Yes",DialogInterface.OnClickListener{dialog, i ->
                Toast.makeText(this@MainActivity, "Positive", Toast.LENGTH_LONG).show()
            })
            setNegativeButton("No",DialogInterface.OnClickListener { dialog, i ->
                Toast.makeText(this@MainActivity,"nagtive",Toast.LENGTH_LONG).show()
            })
        }
        val dialog  = builder.create()
        dialog.show()
    }

//    private fun replace(fr : Fragment){
//        val transition = supportFragmentManager.beginTransaction()
//        transition.replace(androidx.fragment.R.id.fragment_container_view_tag ,fr)
//        transition.commit()
//    }
//    private fun addnaf(){
//        binding.navigationBarItemLabelsGroup.setOnItemSelectedListener{item ->
//            when(item.itemId){
//                R.id.icon1 ->{
//                    replace(myfragmentone)
//                    true
//                }
//                R.id.icon2 ->{
//                    replace(myfragmenttwo)
//                    true
//                }
//
//                else -> {
//                    false
//                }
//            }
//        }
//    }

//private fun initSpinner(){
//    var item = listOf("A","B","C","D")
//    val mayAdapter = ArrayAdapter(this@MainActivity,R.layout.spinner_layout,item)
//
//    binding.spinner.apply {
//        adapter = mayAdapter
//        onItemSelectedListener = object : AdapterView.OnItemSelectedListener{
//            override fun onItemSelected(
//                parent: AdapterView<*>?,
//                view: View?,
//                position: Int,
//                id: Long
//            ) {
//                Toast.makeText(this@MainActivity , item[position] ,Toast.LENGTH_SHORT).show()
//            }
//
//            override fun onNothingSelected(parent: AdapterView<*>?) {
//            }
//
//        }
//    }
}





